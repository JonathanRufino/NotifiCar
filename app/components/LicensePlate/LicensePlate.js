import React from 'react';
import { View, Text } from 'react-native';

import styles from './styles';

const LicensePlate = ({ licensePlate }) => (
    <View style={styles.container}>
        <Text style={styles.text}>
            { licensePlate.toUpperCase() }
        </Text>
    </View>
);

export default LicensePlate;
