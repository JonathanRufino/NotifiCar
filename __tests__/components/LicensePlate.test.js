import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import LicensePlate from '../../app/components/LicensePlate';

describe('<LicensePlate />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <LicensePlate
                licensePlate='ABC-1234'
            />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
