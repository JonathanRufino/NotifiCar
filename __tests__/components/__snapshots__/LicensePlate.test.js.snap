// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LicensePlate /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#d6d8db",
      "borderColor": "#aeb2b7",
      "borderRadius": 10,
      "borderWidth": 5,
      "height": 75,
      "justifyContent": "center",
      "marginVertical": 10,
      "width": 275,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#000",
        "fontFamily": "Mandatory",
        "fontSize": 50,
      }
    }
  >
    ABC-1234
  </Text>
</View>
`;
